pyperf/tests/test_perf_cli.py

```diff
@@ -154,6 +154,26 @@ def test_compare_to_table(self):
         self.assertEqual(stdout.rstrip(),
                          expected)
 
+
+    def test_compare_to_md_table(self):
+        ref_result = self.create_bench((1.0,),
+                                       metadata={'name': 'telco'})
+
+        changed_result = self.create_bench((2.0,),
+                                           metadata={'name': 'telco'})
+
+        stdout = self.compare('compare_to', ref_result, changed_result, '--table',
+                              '--table-format', 'md')
+
+        expected = textwrap.dedent('''
+            | Benchmark | ref      | changed                |
+            |-----------|:--------:|:----------------------:|
```

Comment:
Does it work without spaces? In some MarkDown examples, I saw:

```suggestion
            | --------- | :------: | :--------------------: |
```

Reply:
> Does it work without spaces? In some MarkDown examples, I saw:

yes it works.
see https://gist.github.com/corona10/6db6701efdc4769c237d3d3de5342489

Reply:
I like centered values, it's nice :-)